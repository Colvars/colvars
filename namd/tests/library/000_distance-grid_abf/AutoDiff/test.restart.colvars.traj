# step         one                   fa_one                
          20    3.19500142883077e+00 -2.00127627059354e+00 
          21    3.19091235926421e+00 -1.83655678093969e+00 
          22    3.18689102877202e+00 -2.02779471927389e+00 
          23    3.18304398646448e+00 -2.50868601661760e+00 
          24    3.17946034172504e+00 -3.20793288196262e+00 
          25    3.17620810295805e+00 -4.05360363219171e+00 
          26    3.17333232282842e+00 -4.97767855224492e+00 
          27    3.17085517082510e+00 -5.92001161542199e+00 
          28    3.16877768272762e+00 -6.83099319137926e+00 
          29    3.16708266056064e+00 -7.67258384905857e+00 
          30    3.16573827082641e+00 -8.41821679971556e+00 
          31    3.16470189541734e+00 -9.05180805822893e+00 
          32    3.16392411295527e+00 -9.56669442294402e+00 
          33    3.16335264560683e+00 -9.96442501653096e+00 
          34    3.16293610819235e+00 -1.02534301892968e+01 
          35    3.16262726171125e+00 -1.04473359016060e+01 
          36    3.16238550246452e+00 -1.05630326492936e+01 
          37    3.16217843220556e+00 -1.06187381167053e+01 
          38    3.16198265895496e+00 -1.06325725682142e+01 
          39    3.16178405684152e+00 -1.06216993704827e+01 
          40    3.16157768320120e+00 -1.06018969033008e+01 
