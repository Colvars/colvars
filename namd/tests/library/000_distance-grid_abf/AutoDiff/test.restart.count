# 1
#  0.00000000000000e+00  5.00000000000000e-01         20  0

  2.50000000000000e-01                      0
  7.50000000000000e-01                      0
  1.25000000000000e+00                      0
  1.75000000000000e+00                      0
  2.25000000000000e+00                      0
  2.75000000000000e+00                      0
  3.25000000000000e+00                     40
  3.75000000000000e+00                      0
  4.25000000000000e+00                      0
  4.75000000000000e+00                      0
  5.25000000000000e+00                      0
  5.75000000000000e+00                      0
  6.25000000000000e+00                      0
  6.75000000000000e+00                      0
  7.25000000000000e+00                      0
  7.75000000000000e+00                      0
  8.25000000000000e+00                      0
  8.75000000000000e+00                      0
  9.25000000000000e+00                      0
  9.75000000000000e+00                      0
