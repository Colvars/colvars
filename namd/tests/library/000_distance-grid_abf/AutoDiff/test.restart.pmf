# 1
#      -0.25       0.5        21  0

 0  5.30095
 0.5  5.30095
 1  5.30095
 1.5  5.30095
 2  5.30095
 2.5  5.30095
 3  5.30095
 3.5  0
 4  0
 4.5  0
 5  0
 5.5  0
 6  0
 6.5  0
 7  0
 7.5  0
 8  0
 8.5  0
 9  0
 9.5  0
 10  0
