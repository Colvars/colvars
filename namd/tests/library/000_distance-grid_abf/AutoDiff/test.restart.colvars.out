colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 10
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: ----------------------------------------------------------------------
colvars: Restarting from file "test.colvars.state".
colvars:   Restarting collective variable "one" from value: 3.195
colvars:   Restarting abf bias "abf1" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Opening trajectory file "test.restart.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars:   Prepared sample and gradient buffers at step 20.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
